package.json

```diff
@@ -23,6 +23,7 @@
     "react-dom": "~16.2.0",
     "react-flexbox-grid": "~2.0.0",
     "react-fontawesome": "~1.6.1",
+    "react-highlighter": "^0.4.2",
```

Comment:
small dependency which should be fine.

Reply:
hope so, 15.000 downloads a week should be enough :D anyway, is the style of the highlighting ok or should I change something there?